src/diagonal.works/b6/ingest/yaml.go

```diff
@@ -85,16 +85,52 @@ func (r *RelationMemberYAML) UnmarshalYAML(unmarshal func(interface{}) error) er
 	return nil
 }
 
+type CollectionYAML struct {
+	Keys   []InterfaceYAML
+	Values []InterfaceYAML
+}
+
+type InterfaceYAML struct {
+	v interface{}
+}
+
+func (i InterfaceYAML) MarshalYAML() (interface{}, error) {
+	switch v := i.v.(type) {
+	case b6.FeatureID:
+		return map[string]interface{}{"v": FeatureIDYAML{FeatureID: v}}, nil
+	default:
```

Comment:
i should do path/point/area special yaml parsing too right ? like featureid
and then ok to leave interface as is for anything else (numbers / tags) ?

Reply:
Yes, but I'd leave it like this for the moment, because I need to implement arbitrary value import/export for code, so I'll use it here too, so everything's consistant.

Reply:
lovely